ros/src/computing/perception/detection/packages/lidar_tracker/nodes/euclidean_cluster/euclidean_cluster.cpp

```diff
@@ -25,6 +26,7 @@
 #include <pcl/segmentation/sac_segmentation.h>
 #include <pcl/segmentation/extract_clusters.h>
 #include <pcl/segmentation/conditional_euclidean_clustering.h>
+#include <pcl/segmentation/progressive_morphological_filter.h>
```

Comment:
This file is not included at the ubuntu 14.04 (trusty).

I think that it should keep compatibility with the current recommended environment for a while.

Please keep compatibility with 14.04 by switching the function depending on version of distribution.